cmd/network-console-collector/resources/static/prometheus.yml

```diff
@@ -0,0 +1,11 @@
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+scrape_configs:
+  - job_name: "prometheus"
+    metrics_path: "/metrics"
+    static_configs:
+      - targets: ["network-console:8080"]
```

Comment:
Name resolution won't be choice on non-kube sites as well.

This could also be generated as part of the bootstrap process
determining the port dynamically based on availability, or we
could assume 127.0.0.1:8080 instead at first.